dev/tasks/propose-tag

```diff
@@ -103,6 +103,10 @@ for channeldir in channels autopilot-channels; do
   find ${REPO_ROOT}/operator/${channeldir}/packages -type f -name "*.yaml" | xargs sed -i -e "s@gcr.io/cnrm-eap/@gcr.io/gke-release/@g"
 done
 
+# Update operator RBAC.  
+cp ${REPO_ROOT}/config/installbundle/components/clusterroles/cnrm_viewer.yaml ${REPO_ROOT}/operator/config/rbac/cnrm_viewer_patch.yaml
+
```

Comment:
This step is equivalent to the legacy release "copy-dependency-manifests" https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/operator/Makefile#L105, which is a very bad practice of using [go script to build a `ClusterRole` object on the fly](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/operator/scripts/copy-dependency-manifests/main.go#L98), and made a series of changes just to establish a simple kustomize patch.

This step is missed in the new KCC release and causes CCC lack RBAC for new groups.